databases/week1/hyf_lesson1_Homework.sql

```diff
@@ -0,0 +1,81 @@
+/* Find out how many tasks are in the task table */
+select
+  COUNT(id)
+from
+  task
+  /* Find out how many tasks in the task table do not have a valid due date */
+select
+  COUNT(id)
+from
+  task
+where
+  IsNull(due_date)
+  /* Find all the tasks that are marked as done */
+select
+  *
+from
+  task
+where
+  status_id = 3
```

Comment:
Instead of using the status id, prefer to join the status table and use the name column there.

The reason for this is that it makes it easier to understand, just by looking at your query, what you are trying to do.

If the status table did not have an index on status_id and it was a big table, then you should prefer to do what you did, but in this case it is a very small table, and it doesnt seem like the kind of table that would get big over time

Reply:
okay .. thanks for advise